### PR TITLE
Label colours are black.

### LINKED ISSUE #382: Some axes labels are gray.
```
	${EXPERIMENT_REPO}/bin/plot_krun_results --export-size ${WIDTH_2COL},5 --with-changepoint-means --with-outliers -o examples/changepoint_example.pdf -b bencher7:richards:Hotspot:default-java:14 ${BENCHER7_DATA} --core-cycles 0,1,2 --inset-xlimits 0,9
```

gray

### PR DESCRIPTION
**Pull request by snim2**
Fixes #382 

Please test, can be merged as-is.

### CODE REVIEW

**Comment by vext01** [comment]
The axes are fixed, but I notice some other potential shading issues:

shading

Shouldn't the plot on the right be all gray?

Invocation:
```
	${EXPERIMENT_REPO}/bin/plot_krun_results --export-size 6,4 --no-zoom -o examples/new_inconsistent.pdf -b bencher6:fannkuch_redux:LuaJIT:default-lua:24 -b bencher6:fannkuch_redux:LuaJIT:default-lua:9 --wallclock-only --with-changepoint-means --with-outliers ${BENCHER6_DATA}
```

**Comment by snim2** [comment]
Why should the plot on the right be gray, it has a steady state?

**Comment by vext01** [comment]
Ah, my mistake, I thought slowdown would also be gray.

I think we are good then. Ready to merge?

**Comment by snim2** [comment]
Hang on a bit, still testing

**Comment by snim2** [comment]
These commits don't need squashing, but please do test again before merging.

File: bin/plot_krun_results [hunk 1]
```diff
@@ -1237,7 +1237,7 @@ def get_data_dictionaries(json_files, benchmarks=[], wallclock_only=False,
                         if changepoints:
                             data_dictionary['changepoints'][key][machine].append(data['changepoints'][key][p_exec])
                             data_dictionary['changepoint_means'][key][machine].append(data['changepoint_means'][key][p_exec])
-                            data_dictionary['changepoint_vars'][key][machine].append(data['changepoint_means'][key][p_exec])
+                            data_dictionary['changepoint_vars'][key][machine].append(data['changepoint_vars'][key][p_exec])
```
Comment (vext01) [inline bin/plot_krun_results:1240]: ouch!

**Comment by vext01** [comment]
I've not noticed anything dodgy in my experimentation. Shall I marge?

**Comment by snim2** [comment]
Yes please!

**Review by vext01 — APPROVED** [review]

**Pull request merged**